vhdbuilder/packer/configure-windows-vhd.ps1

```diff
@@ -35,6 +35,30 @@ function DownloadFileWithRetry {
     }
 }
 
+function DownloadFileWithAzCopy {
+    param (
+        $URL,
+        $Dest
+    )
+
+    if (![System.IO.File]::Exists(".\azcopy")) {
+        Write-Log "Downloading azcopy"
+        Invoke-WebRequest -UseBasicParsing "https://aka.ms/downloadazcopy-v10-windows" -OutFile azcopy.zip
+        Expand-Archive -Path azcopy.zip -DestinationPath ".\azcopy" -Force
+        $env:AZCOPY_AUTO_LOGIN_TYPE="MSI"
```

Comment:
The environment variables may not exist after a reboot but the files should be still there so it will skip setting the environment variables.

Reply:
Will remove the setting of environment variables out of the if scope.